spec/index.bs

```diff
@@ -14782,6 +14867,15 @@ This feature adds no [=optional API surfaces=].
 Makes textures with formats {{GPUTextureFormat/"r32float"}}, {{GPUTextureFormat/"rg32float"}}, and
 {{GPUTextureFormat/"rgba32float"}} [=filterable=].
 
+# Behavior Index # {#behavior-index}
+
+Note: Each WGSL [=software extension=] will correspond to a WebGPU [=behavior=].
+
+<h3 id=wgsl data-dfn-type=enum-value data-dfn-for=GPUBehaviorName>`"wgsl"`
```

Comment:
This `id=wgsl` is global. Should use a more specific name like `gpubehaviorname-wgsl`.